pkg/csvcopy/csvcopy.go

```diff
@@ -193,12 +232,16 @@ func (c *Copier) Copy(ctx context.Context, reader io.Reader) (Result, error) {
 		workerWg.Add(1)
 		go func(i int) {
 			defer workerWg.Done()
-			err := c.processBatches(ctx, batchChan)
+			// Add worker ID to context for all operations in this worker
+			workerCtx := WithWorkerID(ctx, i)
```

Comment:
I failed to understand the usefulness of adding worker id into the ctx. Also, why adding worker id into the COPY statements will be useful?

Reply:
1. why adding worker id to the context
Being in the context, it is automatically propagated across interfaces. This means you can implement the  `BatchErrorHandler` and you will still receive the worker id over the context. Then you can use it to log which worker executed the handler.  The main advantage is that if you want, you can forget it exists.

2. why adding worker id to copy statements
I hit a situation where there was a dead lock and the error statements display the entire query. having the id in there allows me to correlate the error with an specific worker. So it is easy for me to link app logs with postgres logs.